### ISSUE TITLE
Await multiple promises

### ISSUE THREAD

**Issue by gitKrystan** [tildeio/ember-async-await-helper#1]
Currently, if you want to await multiple promises, you can nest the promises like so:
```hbs
{{#async-await foo as |foo|}}
  {{#async-await bar as |bar|}}
    {{#async-await baz as |baz|}}
      {{foo}} {{bar}} {{baz}}
    {{/async-await}}
  {{/async-await}}
{{/async-await}}
```

To await multiple promises, it should be possible to do:
```hbs
{{#async-await (hash foo=foo bar=bar baz=baz) as |h|}}
  {{h.foo}} {{h.bar}} {{h.baz}}
{{/async-await}}
```

**Comment by wagenet** [comment]
I'm on it.

**Comment by wagenet** [comment]
See #2

**Comment by gitKrystan** [comment]
Closed by #2
